proto/channel.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package pb;
+
+message Channel {
+    bytes public_key = 1;
+    string name = 2;
```

Comment:
this is different from the name used to make the claim? is this like a title?

Reply:
yes